javascript/treeIntersection/tree-intersection.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const HashMap = require('../hashtable/hashtable');
+
+function treeIntersection(treeOne, treeTwo) {
+  let outputArray = [];
+  if (!treeOne.root || !treeTwo.root) {
+    throw new Error();
```

Comment:
This line is causing an error based on logic from your repeated word hash table that you are calling in your test